src/test/groovy/nebula/plugin/dependencyverifier/DependencyResolutionVerifierTest.groovy

```diff
@@ -46,6 +46,9 @@ class DependencyResolutionVerifierTest extends BaseIntegrationTestKitSpec {
 
         def transitiveNotAvailableDep = new File(mavenrepo.getMavenRepoDir(), "transitive/not/available/a")
         transitiveNotAvailableDep.deleteDir() // to create a missing transitive dependency
+
+        //TODO make task listener changes for config cache
+        disableConfigurationCache()
```

Comment:
I found that we need to take a look at the dependency verifier because I believe is using `TaskExecutionListener` and it is incompatible. While the build does not fail, the listener is not active it appears and the tests don't meet the conditions: an actual build failure